libcudacxx/include/cuda/std/detail/libcxx/include/ctgmath

```diff
@@ -21,12 +21,4 @@
 #include <ccomplex>
 #include "cmath"
```

Comment:
question: do we want to make this header follow a common structure of including config and annotating it as a system header?